apps/obs-web/src/views/DebrieferTrips.vue

```diff
@@ -71,17 +71,39 @@
       </template>
 
       <template v-slot:body="props">
-        <q-tr
-          :props="props"
-          @[computedNativeHover].native="props.selected = !props.selected"
-          @[computedNativeClick].native="props.selected = !props.selected"
-          @keyup.enter.native.stop.prevent="
+        <q-tr :props="props"           
+              @[computedNativeHover].native="props.selected = !props.selected"
+              @[computedNativeClick].native="props.selected = !props.selected"
+              @keyup.enter.native.stop.prevent="
                 $refs.myTable.isRowSelected(props.row.__index)
               "
-          ref="myRef"
-        >
+            ref="myRef">
+
+          <template v-slot:header="props"> 
```

Comment:
@metalsadman I tried adding the header template here but that doesn't do anything . How do I combine the header and body slot in the same code?